.github/workflows/update-engines-version.yml

```diff
@@ -0,0 +1,73 @@
+name: Update Engines Version
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version to check and update the engines version"
+        required: true
+
+jobs:
+  update_engines:
+    name: "Check and update @prisma/engines@${{ github.event.inputs.version }}"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+
+      - uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 6.9.1 
+          
+      # This step uses `@prisma/ensure-npm-release` (abbv. `enr`) https://github.com/prisma/ensure-npm-release
+      - name: Check if version is available on npm
+        run: |
+          echo 'Checking @prisma/engines@${{ github.event.inputs.version }}'
+          pnpx -y -p @prisma/ensure-npm-release enr update -p @prisma/engines -u ${{ github.event.inputs.version }}
+          pnpx -y -p @prisma/ensure-npm-release enr update -p @prisma/engines-version -u ${{ github.event.inputs.version }}
+          echo 'Awesome - proceeding to make the PR'
+       
+      - name: Update the engines
+        run: |
+          echo 'Updating @prisma/engines to ${{ github.event.inputs.version }} using pnpm'
+          cd src
+          pnpm update -r @prisma/engines@${{ github.event.inputs.version }} @prisma/engines-version@${{ github.event.inputs.version  }}
+
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          commit-message: "fix(deps): update engines to ${{ github.event.inputs.version }}"
+          branch: deps/engines
```

Comment:
What happens if we already have a PR open? 

I think this might need to include the inputs.version as well.

Reply:
It matches the renovate behaviour and updates the PR titles and force pushes the removes the earlier commit.

Reply:
Example: https://github.com/pantharshit00/prisma-1/pull/1